### PR TITLE
Add the acceptance list to the PR template

### PR DESCRIPTION
**Pull request by bobbybouwmann**
This makes it possible to add a list of checkboxes to the PR template based on the acceptation criteria defined in the description on Jira.

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -211,20 +215,21 @@ function handlePrCreatePage() {
...
-                        .replace('h2.', '## ')
-                        .replace('h3.', '### ')
-                        .replace('h4.', '#### ')
-                        .replace('h5.', '##### ')
-                        .replace('h6.', '###### ')
-                        .replace('{code}', '```' + NL)
-                        .replace('{{', '``')
-                        .replace('}}', '``');*/
+
+                    var startString = 'h3. Acceptatiecriteria';
```
Comment (RobQuistNL) [inline src/content.js]: I get that the tickets use the word "Acceptatiecriteria" but its Dutch only now. Could you add a set of trigger-words? So we could add English, Dutch, maybe some abbreviations, etc.

**Comment by RobQuistNL** [comment]
Love the idea, love the PR. 1 small request :)

**Comment by bobbybouwmann** [comment]
I updated the search criteria to be configurable by input. They are already set to Dutch by default, but they can be changed now to any string. 

We might need to specify something about the markdown style of Jira?

**Comment by RobQuistNL** [comment]
Love it! Will test this tomorrow and keep you updated.

**Comment by bobbybouwmann** [comment]
@RobQuistNL Did you find some time to test this out?

File: src/content.js [hunk 2]
```diff
@@ -211,20 +221,18 @@ function handlePrCreatePage() {
...
-                        .replace('h2.', '## ')
-                        .replace('h3.', '### ')
-                        .replace('h4.', '#### ')
-                        .replace('h5.', '##### ')
-                        .replace('h6.', '###### ')
-                        .replace('{code}', '```' + NL)
-                        .replace('{{', '``')
-                        .replace('}}', '``');*/
+
+                    description = result.fields.description;
```
Comment (RobQuistNL) [inline src/content.js]: Best case scenario - it works - but this _can_ be ``null``. So, please add a test for if that value is actually a string (or cast it to a string).

Other than that it works great and I think this is a good addition!

Once this is merged I'll bump the version and release a new one to the chrome webstore. Thanks for your effort!

**Pull request merged**